Sparkle/SPUUpdater.m

```diff
@@ -261,6 +266,10 @@ - (NSString *)description { return [NSString stringWithFormat:@"%@ <%@>", [self
 
 - (void)startUpdateCycle
 {
+    if (![self canCheckForUpdates]) {
+        return;
```

Comment:
when would that happen? maybe log a warning?

Reply:
It's intentional behavior. Caller can startUpdater:, and then eg checkForUpdates before startUpdateCycle is called (on next runloop cycle). Once checkForUpdates is invoked, you can't check for updates until that operation is completed. Some comments are in startUpdater: which calls this method.